### DISCUSSION TITLE
Camera Stopped Working and Now Hidden

### DISCUSSION THREAD

**Discussion by nttree** [bitfocus/companion#3184] [Q&A]
Hello Everyone,

I have a JVC PTZ camera that stopped working and is now hidden and I do not know how to 'unhide' it.  I am using Bit companion v3.0.1.  Has this happened with anyone else? Is so, any fixes?

**Comment by Julusian** [comment]
In the top right of the table you can toggle which connections are hidden

![image](https://github.com/user-attachments/assets/4a734098-8a13-4fc7-8270-b18ffacb0117)
